mappings/net/minecraft/util/math/noise/SimplexNoiseSampler.mapping

```diff
@@ -1,6 +1,12 @@
 CLASS net/minecraft/class_3541 net/minecraft/util/math/noise/SimplexNoiseSampler
+	FIELD field_15761 originZ D
+	FIELD field_15762 originY D
+	FIELD field_15763 originX D
 	FIELD field_15764 sqrt3 D
+	FIELD field_15765 permutations [I
 	FIELD field_15766 gradients [[I
+	FIELD field_15767 G2D D
+	FIELD field_15768 F2D D
```

Comment:
What are these?

Reply:
G2 and F2 are constants used in the specification of the simplex noise algorithm, they're not great names of course but it's part of the algorithm

Reply:
You can see them used / explained here http://knielsen-hq.org/simplex_noise_skew_factor.pdf

Reply:
I could make them STRETCH_CONSTANT_2D and SQUISH_CONSTANT_2D

Reply:
(Ignore my deleted comment) I could give them names that describe what they do better

Reply:
Probably `SKEW_FACTOR_2D` and `UNSKEW_FACTOR_2D`

Reply:
They're now `SKEW_FACTOR_2D` and `UNSKEW_FACTOR_2D`